web-gateway/app/controllers/Main.java

```diff
@@ -38,36 +38,4 @@ public Main(Configuration config, MessagesApi messagesApi, UserService userServi
         );
     }
 
```

Comment:
It needs to be injected into the constructor and passed to `super`, but I don't see anywhere the instance variable is read.